examples/pipelines/fraud/resources/apis.yaml

```diff
@@ -1,5 +1,4 @@
 - kind: api
   name: fraud
   model: @dnn
-  compute:
```

Comment:
Good catch! I feel like this should be a separate commit, so I just pushed that to master. Do you mind reverting it here and merging master?